### PR TITLE
fix: ethdubai new dates, add meetup too

### PR DESCRIPTION
**Pull request by patcito**
Small fix to ETHDubai conference date. Also add link to meetup.

### CODE REVIEW

**Review by minimalsm — APPROVED** [review]

File: src/components/MeetupList.js [hunk 1]
```diff
@@ -5,6 +5,12 @@ import Emoji from "./Emoji"
 import Link from "./Link"
 
 const meetups = [
+  {
+    title: "DeFi Dubai",
+    emoji: ":uae:",
+    location: "Dubai",
+    link: "https://www.meetup.com/DeFi-Dubai/",
+  },
```
Comment (minimalsm) [inline src/components/MeetupList.js:13]: Emoji is broke. I _think_ this should fix it?

```suggestion
  {
    title: "DeFi Dubai",
    emoji: ":united_arab_emirates:",
    location: "Dubai",
    link: "https://www.meetup.com/DeFi-Dubai/",
  },
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Hey @patcito, thanks for the PR ❤️🇦🇪 The emoji wasn't working properly, I think using `:united_arab_emirates:` instead of `:uae:` should fix it. 

Could you also move this entry to the bottom of the meetups list as [per our listing criteria here.](https://ethereum.org/en/contributing/adding-products/#product-ordering)

**Comment by minimalsm** [comment]
FYI I've created an issue (#4657) to randomise the meetup list to give all meetups an equal opportunity at visibility :-)

**Comment by patcito** [comment]
I've pushed the changes let me know if this works for you.

**Comment by minimalsm** [comment]
Awesome, thank you. Merging now.

Remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Review by minimalsm — APPROVED** [review]

**Comment by minimalsm** [comment]
@all-contributors please add @patcito for docs

**Pull request merged**